.github/workflows/ci.yml

```diff
@@ -1,17 +1,13 @@
 name: "CI"
 
 on:
-  pull_request:
   push:
-    branches:
-      - "master"
-  schedule:
-    - cron: "42 3 * * *"
```

Comment:
This was originally added in #856 without much discussion and while I agree that it may make sense in some projects, so I don't think it's something that provides many benefits to Ratchet. In particular, we've been receiving a daily build error message for close to two years now. I suggest we remove this for now, but I'm open to introduce this again once we have a clearer use case.

Reply:
Honestly, if the goal is to make sure it keeps working, Renovate and committing the lock file is a better option. But that is something for a different time.